### PR TITLE
Add Groovy to Doc tutorials

### PR DESCRIPTION
**Pull request by lkasso**
Fix https://github.com/gradle/gradle/issues/28484

### CODE REVIEW

**Comment by lkasso** [comment]
@bot-gradle test BD

File: platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc [hunk 1]
```diff
@@ -87,6 +103,36 @@ When you are done with Gradle `init`, the directory should look as follows:
 <4> Build script for `app` subproject
 <5> Default Java source folder for `app` subproject
 <6> Default Java test source folder for `app` subproject
+=====
+[.multi-language-sample]
+=====
+[source, groovy]
+----
+.
+├── gradle                              // <1>
```
Comment (akshayabd) [inline platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc:112]: I think in the latest version we generate a version catalog too. It would be good to show that in the layout. Would you do that in a separate PR?

File: platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc [hunk 2]
```diff
@@ -126,9 +186,9 @@ repositories {
 }
 
 dependencies {
-    testImplementation("org.junit.jupiter:junit-jupiter:5.9.3")     // <3>
+    testImplementation(libs.junit.jupiter)                          // <3>
```
Comment (akshayabd) [inline platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc:189]: Here the version catalog is being referenced but I dont see it in the layout above

File: platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc [hunk 3]
```diff
@@ -152,6 +212,47 @@ tasks.named<Test>("test") {
...
+plugins {
+    id 'application'                                                // <1>
+}
+
+repositories {
+    mavenCentral()                                                  // <2>
+}
+
+dependencies {
+    testImplementation libs.junit.jupiter                           // <3>
```
Comment (akshayabd) [inline platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc:229]: Same about the version catalog here

File: platforms/documentation/docs/src/docs/userguide/running-builds/tutorial/part1_gradle_init.adoc [hunk 4]
```diff
@@ -92,6 +108,32 @@ When you are done, the directory should look as follows:
...
 <7> Some additional Git files may be present such as `.gitignore`.
+=====
+[.multi-language-sample]
+=====
+[source, groovy]
+----
+.
+├── .gradle             // <1>
+│   └── ⋮
+├── gradle              // <2>
```
Comment (akshayabd) [inline platforms/documentation/docs/src/docs/userguide/running-builds/tutorial/part1_gradle_init.adoc:119]: Same question about the version catalog here
